host-interaction/file-system/copy/copy-file.yml

```diff
@@ -7,7 +7,7 @@ rule:
       - michael.hunhoff@mandiant.com
     scopes:
       static: function
-      dynamic: thread
+      dynamic: call
```

Comment:
our rule parser doesn't like a call subscope in a file-wide call scope: `call subscope supported only for the process and thread scopes` - that's why the rule was as is, I assume

Reply:
this works here:
```yml
rule:
  meta:
    name: copy file
    namespace: host-interaction/file-system/copy
    authors:
      - moritz.raabe@mandiant.com
      - michael.hunhoff@mandiant.com
    scopes:
      static: function
      dynamic: call
    mbc:
      - File System::Copy File [C0045]
    examples:
      - Practical Malware Analysis Lab 01-01.exe_:0x401440
  features:
    - or:
      - api: kernel32.CopyFile
      - api: kernel32.CopyFileEx
      - api: CopyFile2
      - api: CopyFileTransacted
      - api: LZCopy
      - api: System.IO.FileInfo::CopyTo
      - api: System.IO.File::Copy
      # static
      - basic block:
        - and:
          - number: 2 = FO_COPY
          - or:
            - api: kernel32.SHFileOperation
      # dynamic
      - and:
        - number: 2 = FO_COPY
        - or:
          - api: kernel32.SHFileOperation

```

Reply:
Oh, yes, that's not what we want.
I think we'd have to change the rule parser to properly handle this. @williballenthin, what do you think?

Reply:
This makes sense to me. +1

Reply:
https://github.com/mandiant/capa/pull/2490

Reply:
https://github.com/mandiant/capa/pull/2490 merged, so we can create rules with call/call scopes now.